node/core/dispute-coordinator/src/metrics.rs

```diff
@@ -163,6 +203,34 @@ impl metrics::Metrics for Metrics {
 				))?,
 				registry,
 			)?,
+			participation_durations: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_dispute_participation_durations",
+						"Time spent within fn Participation::participate",
```

Comment:
Are we sure that default buckets are useful? What kind of values do we expect?

Reply:
[The default buckets](https://docs.rs/prometheus/0.13.3/src/prometheus/histogram.rs.html#25-27) might not be useful for example if all values go into a single bucket for example: `5.0`. We need to know how the values look like for happy case scenarios at least and it really depends on the usecase for the metric. It is for example very useful to observe performance degradations or improvements with very granular buckets.

Reply:
To get a baseline measurement I'd like to use the default buckets at first. Then if we don't have the necessary specificity I'll at least have an idea about what custom bucket size to assign. I'll make a separate PR with changed buckets if necessary.

Reply:
@sandreim I've put together a dashboard providing a view of these new metrics. https://grafana.parity-mgmt.parity.io/d/LAbhoE-Vz/new-dispute-participation-metrics?orgId=1&var-data_source=thanos&var-chain=versi_v1_10&from=now-2d&to=now
 Could use input on which bucket sizes may be best for participation. Also it seems like something may be wrong with the way the metric `polkadot_parachain_dispute_participation_pipeline_durations_bucket` is being handled. 

The time taken for the full pipeline is on average than the time taken for just the participation job, which shouldn't be the case. Mind taking a look at the dashboard to see if these metrics are making sense?